client/api/eventsApi.js

```diff
@@ -21,6 +21,7 @@ export function showAllEvents () {
 }
 
 export function joinEvent (id, eventID) {
+  console.log('request')
```

Comment:
💭 Double-check in the network tab in the browser devtools to see if the correct authorisation header is being sent in the request.  It may also be helpful to check the contents of the bearer token via `console.log(getDecodedToken())`.

Reply:
SOLVED!!! yes!!!